mlrun/runtimes/function.py

```diff
@@ -412,6 +450,10 @@ def with_http(
         return self
 
     def add_model(self, name, model_path, **kw):
+        warnings.warn(
+            'This method is deprecated. Use the "serving" runtime instead',
+            PendingDeprecationWarning,
+        )
```

Comment:
Do you think anyone really still uses this method ? if not it can be changed to `DeprecationWarning`
If it's `DeprecationWarning` we usually include a clear timeline - like "bla bla bla... This will be removed in 0.9.0"
If it's `PendingDeprecationWarning` we usually add a `TODO` of the timeline - when to move to deprecated (and usually that's when we change demos & examples, and when to remove it